### PR TITLE
Drop Class Based Views

### LINKED ISSUE #885: Drop class-based views
* change class based views to 'classic' 
* no need to change redirect views

### PR DESCRIPTION
**Pull request by szewczykmira**
Closes #885

### CODE REVIEW

File: saleor/dashboard/order/views.py [hunk 1]
```diff
@@ -1,34 +1,50 @@
...
+@staff_member_required
+def order_list(request):
+    orders = Order.objects.prefetch_related(
+        'groups', 'payments', 'groups__items').all()
+
+    active_status = request.GET.get('status')
+    if active_status:
+        orders = orders.filter(status=active_status)
+
+    paginator = Paginator(orders, 20)
```
Comment (mociepka) [inline saleor/dashboard/order/views.py]: We have helper method for that already:
https://github.com/mirumee/saleor/blob/master/saleor/core/utils/__init__.py#L73

Comment (patrys) [inline saleor/dashboard/order/views.py]: Shouldn't we also fix that one? (See my comment above.)

**Comment by auvipy** [comment]
may I know the reasons of dropping class based views? did you considered django-vanilla-views?

**Comment by patrys** [comment]
@auvipy we don't use them anywhere else so we're getting rid of the remaining few. Saleor contains very few views that could benefit from an abstract generic base and our own views are highly specialized and meant to be modified directly rather than subclassed so we're not planning to write any generic views ourselves.

File: saleor/core/utils/__init__.py [hunk 2]
```diff
@@ -70,14 +72,24 @@ def get_currency_for_country(country):
...
-        items = paginator.page(page)
-    except PageNotAnInteger:
-        items = paginator.page(1)
-    except EmptyPage:
-        items = paginator.page(paginator.num_pages)
+        page_number = int(page_number)
+    except ValueError:
+        raise Http404(pgettext_lazy(
+            'Message related to an pagination',
+            'Page can not be converted to an int.'))
```
Comment (patrys) [inline saleor/core/utils/__init__.py]: I don't think `Http404` are visible to users? There's not reason to translate system exceptions.

File: saleor/core/utils/__init__.py [hunk 3]
```diff
@@ -6,8 +6,10 @@
 from babel.numbers import get_territory_currencies
 from django import forms
 from django.conf import settings
-from django.core.paginator import EmptyPage, PageNotAnInteger, Paginator
+from django.core.paginator import Paginator, InvalidPage
```
Comment (patrys) [inline saleor/core/utils/__init__.py]: Please use `isort`

**Pull request merged**